### PR TITLE
CNV16551: Adding 4.10 release note for tech preview of OADP integration for backup/restore

### PR DESCRIPTION
**Pull request by mgarrell777**
This PR is for a 4.10 release note story [CNV-16551](https://issues.redhat.com/browse/CNV-16551) about the Tech Preview for the backup and restore functions of OADP integration.

Preview: Bullet starting with "A cluster administrator can now" in https://deploy-preview-42557--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-10-release-notes#virt-4-10-technology-preview

### CODE REVIEW

**Review by aglitke — APPROVED** [review]
looks good

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -117,6 +117,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 * {VirtProductName} has xref:../virt/logging_events_monitoring/virt-virtualization-alerts.adoc#virt-virtualization-alerts[critical alerts] that inform you when a problem occurs that requires immediate attention. Now, each alert has a corresponding description of the problem, a reason for why the alert is occurring, a troubleshooting process to diagnose the source of the problem, and steps for resolving the alert.
 
+* A cluster administrator can now back up namespaces containing VMs by using the xref:../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[OpenShift API for Data Protection] with the `kubevirt` plugin.
```
Comment (ousleyp) [inline virt/virt-4-10-release-notes.adoc:120]: s/containing/that contain

s/plugin/plug-in

(IBM Style Guide)
plug-in   noun, adjective
Write as shown: hyphenated.

plugin   noun, adjective
Do not use. Use "plug-in".

**Review by ousleyp — COMMENTED** [review]
Two tiny nits but overall this LGTM!

**Pull request merged**